python/tests/kat/t_active_health_check.py

```diff
@@ -0,0 +1,140 @@
+import time
+from typing import Generator, Tuple, Union
+
+from abstract_tests import AmbassadorTest, HealthCheckServer, Node
+from kat.harness import Query
+
+"2022-10-04T23:24:45Z"
+"2022-10-04T23:26:45Z"
+
+
+class ActiveHealthCheckTest(AmbassadorTest):
+    def init(self):
+        self.target = HealthCheckServer()
+
+    def config(self) -> Generator[Union[str, Tuple[Node, str]], None, None]:
```

Comment:
We should deprecate the usage of this because it uses annotations to apply the yaml vs using the `def manifest` function which will actually play yamls directly. It doesn't have to be apart of this PR but just something to keep in mind.